modules/enabling-OVS-balance-slb-mode.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// IPI
+// * installing/installing_bare_metal/ipi/ipi-install-installation-workflow.adoc
+// * installing/installing_bare_metal/ipi/ipi-install-post-installation-configuration.adoc
+// UPI
+// * installing/installing_bare_metal/upi/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/upi/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/upi/installing-bare-metal.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-OVS-balance-slb-mode_{context}"]
+= Enabling OVS balance-slb mode for your cluster
+
+You can enable the Open vSwitch (OVS) `balance-slb` mode for the infrastructure for your cluster so that two or more physical interfaces can share their network traffic. A `balance-slb` mode interface provides source load balancing (SLB) capabilities for your cluster, where the interface can act independently without needing to communicate with a network switch. Source load balancing works by assigning a Media Access Control (MAC) address and a virtual local area network (vLAN), if required, to a physical interface, such as `br-phy`. The physcial interface connects to a bridge uplink switch that operates as a port to receive packets from a physical interface and then transmit the package to the `br-ex` interface that are on the same network.
+
+You can integrate the `balance-slb` mode interface into the following network types:
```

Comment:
I'm not sure we need this. What we're documenting here is how to use balance-slb for br-ex. The secondary and localnet networks attach the same way they do in any other architecture.

I guess it doesn't hurt to mention it here, but given how complex the rest of this is I'm not sure we need to.